src/index.js

```diff
@@ -32,9 +33,10 @@ var Time = React.createClass({
     var machineReadable = value.format('YYYY-MM-DDTHH:mm:ssZ');
 
     if (relative || format) {
+      var titleValue = value.format(titleFormat || 'YYYY-MM-DD HH:mm');
```

Comment:
Default `titleFormat` value is better be defined via `getDefaultProps`, I think.